integration/projection_array_test.go

```diff
@@ -0,0 +1,210 @@
+package integration
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func TestProjectionQuerySlice(t *testing.T) {
```

Comment:
I think this test should be in `query_projection_test.go`. And it should be called `TestQueryProjectionSlice`.

Reply:
I suggest renaming the file also, to be in the same style as in other places, i.e `to query_slice_test.go`

Reply:
We already have `query_projection_test.go`

Reply:
@w84thesun @seeforschauer so which one is it? 😄

Reply:
I guess I'm either for @w84thesun's suggestion (I agree that there aren't so many tests for `$slice` that it deserves it's own file) or for storing document projection tests and array projection tests in separate files; @seeforschauer WDYT?

Reply:
it's ok to store it in the existing file.